src/components/alert/index.tsx

```diff
@@ -1,20 +1,40 @@
 import {ReactNode} from 'react';
+import {
+  CheckCircledIcon,
+  ExclamationTriangleIcon,
+  InfoCircledIcon,
+} from '@radix-ui/react-icons';
 
 // explicitly not usig CSS modules here
 // because there's some prerendered content that depends on these exact class names
 import './styles.scss';
 
 type AlertProps = {
   children?: ReactNode;
-  level?: 'info' | 'warning';
+  level?: 'info' | 'warning' | 'success';
   title?: string;
 };
 
+const ICON_MAP = {
+  info: InfoCircledIcon,
+  warning: ExclamationTriangleIcon,
+  success: CheckCircledIcon,
+} as const;
+
 export function Alert({title, children, level = 'info'}: AlertProps) {
+  const Icon = ICON_MAP[level];
+
+  if (!Icon) {
+    throw new Error(`Invalid alert level: "${level}" passed to Alert component`);
```

Comment:
@chargome what is your opinion on this? I opted to error out if an invalid level is passed in. this way, you'll know and the build will fail if you do this.

Alternatively, we would have to check the level and fall back to `info` if something invalid is passed in 🤔 I _think_ it is more helpful to error there, as we def. had cases where users passed in invalid stuff and possibly did not even notice it. But happy to defer to your POV on this here!